lte/gateway/deploy/roles/ovs_prepare/tasks/main.yml

```diff
@@ -124,7 +124,7 @@
       - "{{ all_vars.patches }}"
 
 - name: Cherry pick vlan patch
-  command: git cherry-pick "{{ all_vars.vlan_fix_hash }}"
+  command: git apply -3 "{{ all_vars.vlan_fix_hash }}"
```

Comment:
this should probably be something like `bash -c 'git show "{{ all_vars.vlan_fix_hash }}" | git apply"'`